src/main/java/Counter.java

```diff
@@ -0,0 +1,18 @@
+public class Counter {
+
+    private final AppLabel label;
+    private int value;
+
+    public Counter(AppLabel label) {
+        this.label = label;
+    }
+
+    public void updateValue() {
```

Comment:
Do you want to try writing a test for this? that value gets increased by one when the method is called?

Reply:
I made a first attempt but the test failed, not sure why. I'll try again now